tests/console/check_package_version.pm

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Check package version only for sles12sp5 migration scenarios
+# Maintainer: Yutao Wang <yuwang@suse.com>
+
+use strict;
+use warnings;
+use version;
+use base 'installbasetest';
+use testapi;
+use utils 'systemctl', 'zypper_call';
+
+my @packages = qw(autofs openscap augeas freeradius-server gpgme libgpg-error0); #apache2-mod_auth_openidc);
```

Comment:
Please add the apache2-mod_auth_openidc to package list, or remove it if not needed.